libs/components/src/lib/Accordion/index.tsx

```diff
@@ -0,0 +1,75 @@
+import cls from "classnames";
+import { FC, ReactNode, useState } from "react";
+import { PlusIcon } from "../Icons/plus";
+import classes from "./index.module.scss";
+
+interface AccordionItemProps {
+  title: string;
+  children: ReactNode;
+}
+
+const AccordionItem: FC<AccordionItemProps> = ({ title, children }) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
index file must be contained just jsx codes, move state or functions to index.hook file

Reply:
move useState to new index.hook.ts file